src/flick/docker-compose.server.yml

```diff
@@ -24,11 +24,13 @@ services:
         restart: always
         volumes:
             - static:/static
+            - /home/alanna/docker-compose/aps.pem:/usr/src/app/aps.pem
```

Comment:
on the server, our `docker-compose.yml` is living in `/home/alanna/docker-compose/`